phpstan.neon

```diff
@@ -19,3 +22,5 @@ parameters:
 		- '#Method PclZip::extract\(\) invoked with #'
 		- '#overridden property WP_List_Table::\$items#'
 		- '#Method WPBDP_[a-zA-Z0-9\\_]+::[a-zA-Z0-9\\_]+\(\) is unused#'
+		- '#Call to function is_wp_error\(\) with array will always evaluate to false#'
+		- '#function wp_update_post expects array#'
```

Comment:
Can we fix this error instead of excluding them?

It looks like the wp_update_post line causing the error should be:
```
wp_update_post(
array(
'ID' => $img->ID, 
'post_excerpt' => $value[1],
)
);
```

For the is_wp_error error, I'm not sure why it's having trouble on that one row. Let's exclude the error in that one spot instead of allowing it globally.
https://phpstan.org/user-guide/ignoring-errors#ignoring-in-code-using-phpdocs